src/overlays/actors/ovl_En_Zow/z_en_zow.c

```diff
@@ -602,14 +602,14 @@ Gfx* func_80BDDA7C(GraphicsContext* gfxCtx) {
 
 Vec3f D_80BDDD4C = { 400.0f, 0.0f, 0.0f };
 
-void func_80BDDAA0(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+void EnZow_PostLimbDraw1(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
     if (limbIndex == 15) {
         Matrix_MultVec3f(&D_80BDDD4C, &thisx->focus.pos);
     }
 }
 
-void func_80BDDAE0(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx, Gfx** gfx) {
-    func_80BDDAA0(play, limbIndex, dList, rot, thisx);
+void EnZow_PostLimbDraw2(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx, Gfx** gfx) {
```

Comment:
I would probably say
`EnZow_PostLimbDraw2` -> `EnZow_PostLimbDraw`
`EnZow_PostLimbDraw1` -> `EnZow_PostLimbDrawOpa`
since `EnZow_PostLimbDraw1` gets used with the `Opa` Skelanime variant, while `EnZow_PostLimbDraw2` is the more generic one.